src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java

```diff
@@ -59,8 +63,33 @@ public boolean canHandle(
             throw new IllegalStateException("Cannot handle callback");
         }
 
+        AsylumCase asylumCase =
+            callback
+                .getCaseDetails()
+                .getCaseData();
+
+        if (callback.getEvent() == Event.SEND_DIRECTION
+            && getlatestNonStandardRespondentDirection(asylumCase).isEmpty()) {
+
+            return new PreSubmitCallbackResponse<>(asylumCase);
```

Comment:
This can be a part of the canHandle method right? that way you do not have to explicitely return.

Reply:
Moved to canHandle method.